src/services/messengerSync/consumer.js

```diff
@@ -0,0 +1,35 @@
+const { createChannel } = require('../../utils/rabbitmq');
+const { buildAddUserMessage } = require('./utils');
+
+const QUEUE_INTERNAL = Configuration.get('RABBITMQ_MATRIX_QUEUE_INTERNAL');
+const QUEUE_EXTERNAL = Configuration.get('RABBITMQ_MATRIX_QUEUE_EXTERNAL');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Configuration' is not defined.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203703)